src/OpenTelemetry.Shims.OpenTracing/OpenTelemetry.Shims.OpenTracing.csproj

```diff
@@ -0,0 +1,20 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFrameworks>net46;netstandard2.0</TargetFrameworks>
+    <Description>OpenTelemetry shims for OpenTracing</Description>
+    <PackageTags>$(PackageTags);distributed-tracing;OpenTracing</PackageTags>
+    <IsPackable>false</IsPackable>
```

Comment:
I guess it should be packable

Reply:
I agree. I did this intentionally until there was something worth packaging and distributing...